sphinx/users/comlinetools/conversion.rst

```diff
@@ -29,6 +29,17 @@ The output file format is determined by the extension of the output file, e.g.
 
     .. versionadded:: 5.4.0
 
+.. option:: -noflat
+
+    Do not flatten the series i.e. use the sub-resolution API for images
```

Comment:
Perhaps could reword somehow: it's not clear if the "i.e. use the ..." explains the "do not X ..." or the "X ...": it's ambiguous if the "do not" distributes over it.

Reply:
Agreed that it could be misinterpreted and would be best if reworded to remove the "ie" and make it clear if the sub-resolution API is being used or not .